pkg/frontend/mysql_cmd_executor.go

```diff
@@ -1304,10 +1311,24 @@ func (mce *MysqlCmdExecutor) handleLoadData(requestCtx context.Context, proc *pr
 	/*
 		check table
 	*/
-	tableHandler, err := dbHandler.Relation(requestCtx, loadTable)
+	tableHandler, err = dbHandler.Relation(requestCtx, loadTable)
 	if err != nil {
-		//echo client. no such table
-		return moerr.NewNoSuchTable(loadDb, loadTable)
+		txn, err = ses.txnHandler.GetTxn()
+		if err != nil {
+			return err
+		}
+		dbHandler, err = ses.GetStorage().Database(requestCtx, engine.TEMPORARY_DBNAME, txn)
+		if err != nil {
+			return moerr.NewNoSuchTable(loadDb, loadTable)
+		}
+		loadTable = loadDb + "-" + loadTable
```

Comment:
Did you merged previous PR without addressing this '-' problem?

Reply:
This question should be answered by the third and fourth points summarized above. Because the user is not allowed to create "%!%mo_temp_db" database, So we don't have the problems that discussed before. The temporary engine only needs to create a database. Maybe save a little on overhead?

And frankly, it's the least expensive way to change the current code.